tests/simple/testdata/comparisons.textproto

```diff
@@ -721,6 +721,177 @@ section {
   }
 }
 section {
-  name: "Bound comparison"
+  name: "bound"
   description: "Comparing bound variables with literals or other variables"
+  test {
+    name: "self_test_bound_bytes_gt_left_false"
```

Comment:
Omit the 'self_test_bound_' prefix:  all these tests are self-tests, and 'bound' is provided by the section.

Reply:
You're right! I stashed some of the changes before the last PR. Will send it soon.